terminus-ui/tooltip/src/tooltip.component.html

```diff
@@ -1,8 +1,9 @@
-<span
+<span #tooltip="matTooltip"
   class="c-tooltip qa-tooltip"
   [matTooltip]="tooltipValue"
   [matTooltipPosition]="tooltipPosition"
   [ngClass]="{'c-tooltip--underline': hasUnderline}"
+  (click)="tooltip.show()"
```

Comment:
We shouldn't be changing how the tooltip works for consumers. I'd rather add a `show` method to our tooltip component that calls Material's `show`.